unblob/pool.py

```diff
@@ -129,14 +167,16 @@ def submit(self, args):
         self._input.put(args)
 
     def process_until_done(self):
-        while not self._input.is_empty():
-            result = self._output.get()
-            self._result_callback(self, result)
-            self._input.task_done()
+        with contextlib.suppress(EOFError):
```

Comment:
Where do this `EOFError` come from?

Reply:
from multiprocessing's queue, as it is using pipes